mpmp/src/cmds/Whisper.java

```diff
@@ -0,0 +1,43 @@
+package cmds;
+
+import java.util.Arrays;
+import main.Client;
+import main.Conn;
+
+/**
+ * @author Leander
+ */
+public class Whisper implements CmdFunc {
+	private static final String WhisperSyntax = "Syntax: whisper <playername> <message>";
+	
+	@Override
+	public void exec(String line, Conn conn) {
+		String[] args = line.split(" ");
+		if (args.length < 3) {
+			conn.sendErr(WhisperSyntax);
+			return;
+		}
+		
+		Client client = null;
+		for (Client c : Client.getClients())
+			if (args[1].equals(c.getName())) {
```

Comment:
We should place a method to search for a client into the `Client` class instead of exposing the `clients` HashSet to everyone. Please revert commit d5d4e782a99956e3196efe08ec98ca7694e8cacb and add
```java
public static Client search(String name)
```
to `Client` instead. Compare `Cmd.search`.